css/style.css

```diff
@@ -1,11 +1,174 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+ 
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    position: relative;
+}
+ /* Header  */
+ header {
+    font-size: 24px;
+    width: 100%;
+    height: 20px;
+    padding: 40px;
+    margin: 20px;   
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+}
+
+ .logo {
+    width: 100%;
+    height: 50px;
+    align-self: flex-start;
+}
+
+.navbar {
+ }
+
+.navbar ul {
+    display: flex;
+    flex-flow: nowrap;
+    justify-content: flex-end;
+    align-items: center;
+}
+.navbar ul li {
+    list-style-type: none;
+    margin: 10px;
+    padding: 5px;
+
+}
+.navbar ul li a {
+    /* list-style-type: none; */
+    text-decoration: none;
+}
+
+.navbar ul li a:hover {
+    color: orangered;
 }
 
+/* main */
+/* main first section */
+.picture {
+    background-image: url(../img/first-background.jpg);
+    background-position:center;
+    background-size: cover;
+    text-align: center;
+    width: 100%;
+    height: 700px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: white;
+
+}
+.picture-text{}
+
+.picture-text-heading {
+    font-size: 50px;
+}
+
+.picture-text-smalltext {
+    font-size: 30px;
+}
+
+.picture-button {
+    font-size: 24;
+    
+}
+
+.picture-button button {
+    width: 150px;
+    height: 50px;
+    padding: 15px 30px;
+    background-color: orangered;
+    text-align: center;
+    font-size: 24;
```

Comment:
add unit here

Reply:
Also, could you add some rounded corners to follow the design a little more closely?
If you add a border of the same orangered colour as the background, it will hide the current default border as well.

Reply:
I have changed these, thank you.